pkg/kubectl/plugins/plugins.go

```diff
@@ -105,6 +127,16 @@ func (f Flag) Validate() error {
 	if strings.Contains(f.Name, " ") {
 		return ErrInvalidFlagName
 	}
+	if f.Type != "" {
+		if _, ok := flagTypes[f.Type]; !ok {
+			return ErrInvalidFlagType
+		}
+	}
+	if f.Type == BoolFlagType && f.DefValue != "" {
```

Comment:
What if `DefValue` is empty for boolean type?

Reply:
You're right, I'll update the patch so that it defaults to `false` when not specified for boolean flags.